crates/kubelet/src/node.rs

```diff
@@ -249,15 +250,7 @@ fn node_definition(config: &Config, arch: &str) -> serde_json::Value {
         "kind": "Node",
         "metadata": {
             "name": config.node_name,
-            "labels": {
-                "beta.kubernetes.io/arch": arch,
-                "beta.kubernetes.io/os": "linux",
-                "kubernetes.io/arch": arch,
-                "kubernetes.io/os": "linux",
-                "kubernetes.io/hostname": config.hostname,
-                "kubernetes.io/role":     "agent",
-                "type": "krustlet"
-            },
+            "labels": {},
```

Comment:
I personally think it's cleaner to specify the default labels using JSON literal as it did before, instead of all the `to_string` or `to_owned` in https://github.com/deislabs/krustlet/pull/210/files#diff-3c33c7d659ceab610971cced03231c9bR392-R403.

Reply:
I struggled with this decision. On the one hand it's more readable, on the other, you have two different methods managing the same thing.  I don't feel strongly about it one way or the other, so I defaulted to one method for one thing.  That said, if we followed the suggestion you make later on about filtering instead of building, this problem kind of goes away.